src/components/PopularLanguages.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="col m12 l6">
+    <div>
+      <h4>Popular Languages</h4>
+      <span v-if="!popularLanguages.length">No languages :'(</span>
+      <div v-else v-for="(language, index) in popularLanguages" :key="index">
+        <span class="grey-text text-darken-3 rank-position"># {{ index + 1 }}</span>
+        <span class="language-text">{{ language.text }}</span>
+        <strong class="grey-text text-darken-2">{{ language.score }} %</strong>
+        <div class="progress blue lighten-4" style="height: 10px;">
+          <div class="determinate blue darken-2" :style="{ width: `${language.score}%` }"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "PopularLanguages",
+  props: {
+    languages: {
+      type: Array,
+      required: true
+    }
+  },
+  computed: {
+    popularLanguages() {
+      return Array.isArray(this.languages) && this.languages.length
```

Comment:
Since it's already setup the property as type array, isn't the `isArray` validating redundant?

Reply:
You are right.
Fixed. 👍